git-annex-remote-googledrive

```diff
@@ -394,13 +402,15 @@ or re-run 'git-annex-remote-googledrive setup' followed by 'git annex enableremo
         else:
             raise RemoteError ("There are two or more files named {}".format(key))
     
-    def _getfolder(self, path, root=None, create=True):
+    def _getfolder(self, path, root=None, create=True, parent_folder_id='root'):
```

Comment:
I'm still not happy with an additional parameter that does basically the same as `root`. I prefer one of these:
1. Performing the conversion from ID to file object in the code. Eg.:
```
parent_folder_id = self.annex.getconfig('parent_folder_id')
parent_folder = self.drive.CreateFile({'id': parent_folder_id or 'root'})
self.root = self._getfolder(self.prefix, parent_folder)
```
2. Adding a wrapper method. Eg:
```
def _getroot(self, path, parent_folder_id):
    parent_folder = self.drive.CreateFile({'id': parent_folder_id or 'root'})
    return self._getfolder(self.prefix, parent_folder)
```